### ISSUE TITLE
BUG: Don't obtain ZJets from MC

### ISSUE THREAD

**Issue by ram1123** [ram1123/2l2Q_limitSettingTool#77]
As we are obtaining the Zjets from the data-driven way so we should only use the Zjets from data-driven. No, need to read it from MC.

https://github.com/ram1123/2l2Q_limitSettingTool/blob/f35cde480d7c84b14c2e3943b71a1537bae6a4ad/datacardClass.py#L295

**Comment by jialin-guo1** [comment]
@ram1123 https://github.com/ram1123/2l2Q_limitSettingTool/blob/f35cde480d7c84b14c2e3943b71a1537bae6a4ad/datacardClass.py#L366
we don't use MC for Zjets. In 1D templates, I do create ZJets histogram from Alpha Method.

**Comment by ram1123** [comment]
Yes but it’s not what we import into the workspace. Please cross check
